packages/techdocs-cli/src/commands/index.ts

```diff
@@ -100,6 +108,10 @@ export function registerCommands(program: CommanderStatic) {
     .description(
       "Serve a documentation project locally in a Backstage app-like environment"
     )
+    .option(
```

Comment:
Recommend tacking on the `spotify/techdocs` default value to these options declarations for backwards compatibility (believe the third argument is automatically provided as the default value to the underlying command without having to specify it there).

Reply:
Updated